dts/arm/nordic/nrf52840.dtsi

```diff
@@ -55,6 +55,33 @@
 			label = "UART_1";
 		};
 
+		gpiote0: gpiote@40006000 {
+			compatible = "nordic,nrf-gpiote";
+			reg = <0x40006000 0x1000>;
+			status = "disabled";
+			label = "GPIOTE_0";
+		}
+
+		gpio0: gpio@50000000 {
+			compatible = "nordic,nrf5-gpio";
+			gpio-controller;
+			gpiote = &gpiote0;
```

Comment:
What relationship are you trying to enable here?

Reply:
I added it initially due to earlier comments from pizi-nordic, and it seemed like a good idea to express some relationship between the gpio and gpiote. 
However its a bit redundant, so I took it out.

Reply:
Is that configurable? I mean can we configure gpio0 to work with gpiote1?